web/app/login.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Login page</title>
+  </head>
+  <body>
+    <h1>Login</h1>
+    <form action="localhost:5000/login" method="get">
```

Comment:
We can't use form action to call our REST API. We need to use fetch API or AJAX to call backend